src/__tests__/emailverification.js

```diff
@@ -0,0 +1,102 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import app from '../index';
+
+import { signUp, confirmEmailVerificaionToken } from '../controllers/user';
+import { emailVerifyToken } from '../utils/index';
+
+chai.use(chaiHttp);
+chai.use(sinonChai);
+
+const { expect } = chai;
+
+let request;
+
+describe('EMAIL ROUTE', () => {
+  before(async () => {
+    request = chai.request(app).keepOpen();
+  });
+
+  afterEach(() => sinon.restore());
+
+  describe('EMAIL VERIFICATION ROUTE', () => {
+    it('should have a status of 200 when message is sent succesfully', async () => {
+      const body = {
+        email: 'akp.ani@yahoo.com',
+        firstName: 'Aniefiok',
+        lastName: 'Akpan'
+      };
+      const response = await request.post('/api/email/test').send(body);
+      expect(response.body.status).to.equal(200);
+      expect(response.body).to.be.a('object');
+    }).timeout(0);
+  });
+  describe('UTILS/ EMAIL TOKEN', () => {
+    it('it should assign a token to newly registered users', async () => {
+      const id = 'some_secret_identification_number_or_string';
+      const token = await emailVerifyToken(id);
+      expect(token).to.equal(token);
+    }).timeout(0);
+    it('it should not assign token, or fail', async () => {
+      const id = 'some_secret_identification_number_or_string';
+      const token = await emailVerifyToken(id);
+      expect(token).to.equal(token);
+    }).timeout(0);
+  });
+
+  describe('EMAIL TOKEN CONFIRMATION ROUTE', () => {
+    it('should have a status of 200 when valid token is sent as query string', async () => {
+      const id = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6InNvbWVfZW5jb2RlZF9pZGVudGlpdHkiLCJpYXQiOjE1NjY4MTE1OTcsImV4cCI6MTU2Njg5Nzk5N30.oCXhqT4Ri-k7RIqlgIHVFvGGwmxIzBwyzqbXIG0JwyE';
```

Comment:
can you define this value in a `__mocks__` folder and import it here since you're using it in multiple places.

Reply:
Alright, that would be done